circuits/transfer/tests/send_to_contract_obfuscated.rs

```diff
@@ -55,8 +55,13 @@ fn create_random_circuit<R: RngCore + CryptoRng>(
         rng, &c_ssk, &fee, &crossover, &message, &address,
     );
 
-    let message =
-        StcoMessage::new(message, m_r, m_derive_key, m_pk_r, m_blinder);
+    let message = StcoMessage {
+        blinder: m_blinder,
+        derive_key: m_derive_key,
+        message,
+        pk_r: m_pk_r,
+        r: m_r,
+    };
```

Comment:
couldn't use / reuse `blinder` instead of `m_blinder` (and the same for all the `m_` prefixed ones)…?